explainer/integration_with_web_platform.md

```diff
@@ -11,7 +11,7 @@ The origin of the fenced frame will be the regular origin that it got navigated
 The API that generates a fenced frame config can pick the initial size that the fenced frame document sees, subject to whatever restrictions it deems necessary for its privacy model. If the initial size is fixed, then any changes the embedder makes to the width and height attributes of the <fencedframe> will not be reflected inside the fenced frame.
 
 ## Viewability
-Viewability events using APIs like [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) can be a communication channel from the embedding context to the fenced frame. Viewability events are important information in the ads workflow for conversion reports and payments and would be made available in FLEDGE and other ads use cases via separate mechanisms. In the first origin trial of fenced frames, intersection observer will be fully supported and it will be phased out only when the alternative mechanism is launched.
+Viewability events using APIs like [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) can be a communication channel from the embedding context to the fenced frame. Viewability events are important information in the ads workflow for conversion reports and payments and would be made available in Protected Audience and other ads use cases via separate mechanisms. Initially, intersection observer will be fully supported in fenced frames and it will be phased out only when the alternative mechanism is launched. In the meantime, we plan to add metrics to understand honest use and detect abuse, to help guide the development of these eventual mechanisms.
```

Comment:
"Viewability events using APIs like [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) can be used as a communication channel from the embedding context to the fenced frame. However, to be used as a covert channel, it does require movement of the frame containing the ad on the screen and therefore is hostile to user experience.
Viewability events are important information in the ads workflow for conversion reports and billing and therefore, intersection observer will be fully supported in fenced frames and it will be phased out only if/when an alternative mechanism is launched. In the meantime, we plan to add metrics to understand honest use and detect abuse, to help guide the need/development of these eventual mechanisms or mitigations."

Reply:
Let's also remove the contents in the "intersection observer" section in https://github.com/WICG/fenced-frame/blob/master/explainer/README.md and link this section in that point.

Reply:
Done for both